netlify.toml

```
@@ -23,6 +23,8 @@
           "/about/index.html",
           "/contact/index.html",
         ]
+        ignoreElements = '#search'
+        ignoreGuidelines = ['color-contrast']
```

@oluoluoxenfree Instead of ditching the plugin entirely, what do you think about this solution -- selectively ignoring the errors we don't want to address? Seems like we could even switch to `failWithIssues = true` so we get notified about new issues?

---

@jgerigmeyer should i add this to this PR?

---

@oluoluoxenfree Sure! It won't pass until we merge #305 in as well, but that's okay.

---

Do people prefer `failWithIssues`?